VogtPayroll3/PayrollConsoleReader.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll3
+{
+    class PayrollConsoleReader
+    {
+
```

Comment:
Awesome. So here is where you should be building a `var employee = new Employee(field, field, field)`.

OR, if you want to go the extra mile here -- you can separate out the creation of employee and create your first Factory! EmployeeFactory -- which has one method: `CreateEmployee(string employeeName..... etc)` and simply returns the `new Employee(field, field, field)`.